app/requests/base.js

```diff
@@ -34,11 +34,15 @@ module.exports = (constraints, req, res, next) => {
           var response = {
             message: `Validation failed. ${validateJsErrors.length} error(s)`,
           };
+          console.log("validateJsErrors...", validateJsErrors);
 
           let errors = {};
 
           validateJsErrors.map((d) => {
-            errors[d.attribute] = d.error;
+            if (!errors[d.attribute]) {
```

Comment:
@Subbu-betalectic can you please write here and explain to me this piece of code you have implemented?

Reply:
@ShubhamJoshi-1994 In payload if the currency goes empty, We have two validation rules so we are getting two error messages. Earlier we are getting a second error message.

![Screenshot 2022-10-17 at 3 36 07 PM](https://user-images.githubusercontent.com/107619278/196150616-f8817b1e-5d0f-4c98-87eb-82ff218028bc.png)
![Screenshot 2022-10-17 at 3 30 53 PM](https://user-images.githubusercontent.com/107619278/196149453-628cb0c1-0cf7-41ec-95aa-d25be30999f4.png)



now I have added a check to whether the error key is already present in the errors object then it won't overwrite existing error message
![Screenshot 2022-10-17 at 3 42 16 PM](https://user-images.githubusercontent.com/107619278/196151815-c89901a8-c78a-4667-821d-137adca4568a.png)
![Screenshot 2022-10-17 at 3 42 39 PM](https://user-images.githubusercontent.com/107619278/196151886-a5a4955a-5fbe-403c-9bd4-2fad0e8fbb92.png)

Reply:
ok @Subbu-betalectic , it's clear to me now. This should work.